pkg/controller/acicontainersoperator.go

```diff
@@ -241,9 +240,9 @@ func (t *OperatorHandler) ObjectCreated(obj interface{}) {
 	log.Info("OperatorHandler.ObjectCreated")
 
 	acicontainersoperator := obj.(*operators.AciContainersOperator)
-	log.Info(acicontainersoperator.Spec.Config)
+	log.Debug(acicontainersoperator.Spec.Config)
 	if (acicontainersoperator.Spec.Config == ""){
-		log.Info("ACI CNI CR Config is Nil")
+		log.Error("ACI CNI CR Config is Nil")
```

Comment:
nit: can we be consistent with what the actual name of the CRD is? (rather than saying  "ACI CNI CR" which is pretty vague)